landscape/client/monitor/cloudinit.py

```diff
@@ -0,0 +1,69 @@
+import json
+import subprocess
+
+from landscape.client.monitor.plugin import DataWatcher
+
+
+class CloudInit(DataWatcher):
+
+    message_type = "cloud-init"
+    message_key = message_type
+    scope = message_type
+    persist_name = message_type
+    run_immediately = True
+    run_interval = 3600 * 24  # 24h
+
+    def get_data(self) -> str:
+        return json.dumps(get_cloud_init(), sort_keys=True)
+
+    def register(self, monitor):
```

Comment:
You shouldn't need this register. See https://github.com/canonical/landscape-client/pull/130/files and https://github.com/canonical/landscape-client/pull/158

Reply:
Went over offline.